components/applications-service/pkg/server/server.go

```diff
@@ -262,6 +262,35 @@ func (app *ApplicationsServer) GetServices(
 	}, nil
 }
 
+func (app *ApplicationsServer) FindServices(
+	request *applications.ServicesReq,
+	stream applications.ApplicationsService_FindServicesServer) error {
+
+	filters, err := stringutils.FormatFilters(request.GetFilter())
+	if err != nil {
+		return status.Error(codes.InvalidArgument, err.Error())
+	}
+
+	sortField, sortAsc, err := params.GetSortParamsForServices(request.GetSorting())
+	if err != nil {
+		return status.Error(codes.InvalidArgument, err.Error())
+	}
+
+	services, err := app.storageClient.GetServices(sortField, sortAsc, 1, 0, filters)
```

Comment:
I took the approach here of ignoring any incoming pagination params and then disabling pagination in this call. We could instead check for pagination params and return a 400 or possibly even allow pagination with different defaults (though I don't know why you would do that when the unary RPC is already there as an alternative).